### PR TITLE
Restrict pkcs8 accepted key and fix error message

### PR DESCRIPTION
**Pull request by salvatore-coppola**
Fixed private key regex
Fixed apply button event-handling duplication.
Now more runtime exceptions are handled by GwtCertificatesServiceImpl.

### CODE REVIEW

**Comment by salvatore-coppola** [comment]
@nicolatimeus if you want to check

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/security/KeyPairTabUi.java [hunk 1]
```diff
@@ -219,21 +225,42 @@ private void storeKeyPair() {
...
+                            public void onSuccess(Void result) {
+                                reset();
+                                setDirty(false);
+                                KeyPairTabUi.this.listener.onKeystoreChanged();
+                            }
+
+                            @Override
+                            public void onFailure(Throwable caught) {
+                                caught.printStackTrace();
+                                FailureHandler.handle(caught);
```
Comment (nicolatimeus) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/security/KeyPairTabUi.java]: Probably this is not needed

File: kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/security/KeyPairTabUi.java [hunk 2]
```diff
@@ -211,6 +216,7 @@ private void initForm(final List<String> keyStorePids, List<String> usedAliases)
                     storeCertificate();
                 }
                 this.listener.onApply(isValid);
+                this.applyHandler.removeHandler();
```
Comment (nicolatimeus) [inline kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/security/KeyPairTabUi.java]: We should make sure that no handlers are leaked

**Comment by mstankovic** [comment]
verified as implemented.

**Review by nicolatimeus — APPROVED** [review]

**Pull request merged**